### ISSUE TITLE
Hi! Question about calendar.vim

### ISSUE THREAD

**Issue by remembertoremind** [itchyny/calendar.vim#131]
Eventhough I have only been using Linux for a couple of months I'm still a newbie, so please be gentle :) However, this plugin is amazing! :D Thank you soo much!

I have a few suggestions/questions. I might have missed them in the docs but here it goes:
1. I'm not able to add tasks in the calendar. I press "T" add a description, but after I press enter, nothing happens. This is on Arch Linux.

2. I have a Lenovo X250 with a 12" screen. It does not leave much room the view with the 4 day view. To get a better overview it would be nice with less visible hours - I mean like if the calendar is shrinked in height so the hours on the right jumps two hours. I hope this makes sense :)

3. On i3wm I would love to start the vim and the calender with $mod+something. However when i bind the command "exec $terminal -e vim -c "Calendar"" the colors are all funky. Same thing in tmux. Is there any way around this?

4. Is there any way to add notes to my tasks and events - like in calcurse? This is not a dealbreaker, but it would be SO awesome.

**Comment by itchyny** [comment]
Thank you.

1. Press `o` to start adding a new task and `i` or `A` to start editing the existing task.
2. Maybe you mean increasing the height for each hour? I think some screenshot will help me see what's happening.
3. I haven't used i3wm but I cannot reproduce the problem within tmux. Please upload some screenshot and describe how to reproduce.
4. Please refer to `:h calendar-input-format`. The `note: example` syntax is recognized as notes and synced with Google Task when you enable the `g:calendar_google_task` option.

**Comment by remembertoremind** [comment]
Hi again :) Thank you so much for answering my questions. Sorry for the delay. Maybe I should say that I'm using Vim 8.0.1297.

1. I am able to enable the task-overview and I'm able to naming the task, but it never shows up in the overview in the right side of the screen. Moreover, I have added a task in on my google account and it does not show up in calendar.vim. This should be enabled through my vimrc as you suggest :) 
`let g:calendar_google_task = 1`
![2017-12-12-10 32 01-crop](https://user-images.githubusercontent.com/26480902/33877077-3314e5ae-df1f-11e7-8512-8a302cf8f1e6.png)

2. Actually I mean to decrease the hight of each hour. As in, the amount of hours nomatter the hight :) I.e: I would like to view 10 hours of a day, nomatter the hight/width of the screen/window. Atm I have two screens where my 1366x768 is just too small to view a whole day. and my 1920x1080 is great. If I could get 10 or 20 hours in my window nomatter the hight it would be awesome. It would make it far easier the get an overview of the day :) Below is screenshot of two windows. My point is that the smaller window should/could show more of the day by not showing every hour. Or else the user has to scroll a lot. I think this could be a setting like "let g:calendar_hours_day = 15". However, I'm not sure if this is even possible :) Just a suggestion.
![2017-12-12-10 34 13-crop](https://user-images.githubusercontent.com/26480902/33877218-9fbd3f94-df1f-11e7-8002-5fb3c7b199b7.png)

3. I found a solution for i3wm. I just needed to give the right command. My mistake :) However, the tmux-issue persists, but only with some colorschemes. I mostly use the desert colorscheme, and it's not working in tmux. I have attached a screenshot showing a normal urxvt terminal on the left and urxvt-tmux on the right. Both are set to the same colorscheme (desert). I have done nothing special to reproduce. Just started tmux, vim and then loaded the plugin - after that I have tried to set different colorschemes and most of them look like the screenshot. This does not happen with other applications - neither with vim.
![2017-12-12-09 46 33-crop](https://user-images.githubusercontent.com/26480902/33876740-446b000a-df1e-11e7-8c86-d2bc1100045f.png)
 
Thank you :)
